libraries/AP_HAL_ChibiOS/hwdef/common/bouncebuffer.c

```diff
@@ -24,6 +24,9 @@
 #if defined(STM32H7)
 // always use a bouncebuffer on H7, to ensure alignment and padding
 #define IS_DMA_SAFE(addr) false
+#elif defined(STM32F732xx)
+// DTCM DMA does not work on F732
```

Comment:
Here is the comment

Reply:
thanks, I've fixed the comment. I didn't change the implementation as I've already spent a lot of time testing and I'm happy with the results